aggregator/src/aggregator.rs

```diff
@@ -1400,24 +1487,15 @@ impl VdafOps {
                 }
             }
 
-            let hpke_keypair = task
+            let task_hpke_keypair = task
                 .hpke_keys()
-                .get(report_share.encrypted_input_share().config_id())
-                .ok_or_else(|| {
-                    info!(
-                        config_id = %report_share.encrypted_input_share().config_id(),
-                        "Helper encrypted input share references unknown HPKE config ID"
-                    );
-                    aggregate_step_failure_counter.add(
-                        &Context::current(),
-                        1,
-                        &[KeyValue::new("type", "unknown_hpke_config_id")],
-                    );
-                    ReportShareError::HpkeUnknownConfigId
-                });
+                .get(report_share.encrypted_input_share().config_id());
+
+            let global_hpke_keypair =
+                global_hpke_keypairs.keypair(report_share.encrypted_input_share().config_id());
 
             // If decryption fails, then the aggregator MUST fail with error `hpke-decrypt-error`. (§4.4.2.2)
-            let plaintext = hpke_keypair.and_then(|hpke_keypair| {
+            let try_hpke_open = |hpke_keypair: &HpkeKeypair| {
                 hpke::open(
                     hpke_keypair.config(),
                     hpke_keypair.private_key(),
```

Comment:
This isn't quite right, it'll log to info on every decryption trial failure. Let me move to draft and fix that.